src/lib/manageNode.js

```diff
@@ -27,4 +27,54 @@ const startRebuild = async (snapshotServerURL) => {
   });
 };
 
+const setForgingOn = async (nodeSecret) => {
+  const serverStatusExec = `
+    curl -k -H "Content-Type: application/json" -X POST -d '{"secret":"${nodeSecret}"}' http://localhost:7000/api/delegates/forging/enable`;
```

Comment:
Lisk-cute-assistant should work on Mainnet too... Here the url is hardcoded, it should instead be taken from the `config.js`. Please update the URL with the constant and it would be good to use axios for http requests instead of curl.

Ideally we use 'child_process' only for commands that can't be executed on NodeJS and need the operative system.